src/main/java/org/snomed/snowstorm/rest/ConceptController.java

```diff
@@ -485,6 +489,7 @@ public AsyncConceptChangeBatch getConceptBulkChange(@PathVariable String branch,
 			@PathVariable String conceptId,
 			@RequestParam(defaultValue = "inferred") Relationship.CharacteristicType form,
 			@RequestParam(required = false, defaultValue = "false") Boolean includeDescendantCount,
+			@RequestParam(required = false, defaultValue = "") String refsetId,
```

Comment:
Optional parameters should default to `null`, not an empty string, for consistency with the rest of the app please.

Reply:
This parameter name does not seem explicit enough.. it triggers a behaviour where a flag is added to indicate further descendants within a refset.. perhaps this should be called `checkDescendantsWithinRefsetId` ? Please also add some notes to describe the behaviour of this parameter.

Reply:
Both changes made.